pkg/imgpkg/image/dir_image.go

```diff
@@ -42,13 +42,18 @@ func (i *DirImage) AsDirectory() error {
 		return err
 	}
 
+	//TODO: use constants
+	if !strings.Contains(i.dirPath, ".imgpkg/bundles") {
```

Comment:
We talked about the package `image` not being aware of the presence of bundles, so I think we should move this print into the caller of this function.

Reply:
done!